Gemfile

```diff
@@ -38,6 +38,7 @@ gem "connection_pool",                                       :require => false #
 gem "dalli",                            "~>3.2.3",           :require => false
 gem "default_value_for",                "~>4.0"
 gem "docker-api",                       "~>1.33.6",          :require => false
+gem "drb",                              "~>2.2.1",           :require => false
```

Comment:
I'm wondering if we should be more open - maybe ">=2.2.1" or "~>2.2" ?

Reply:
Sure `~>2.2` would work, don't want to accidentally pull in a breaking change